src/main/resources/templates/db_scripts/all_finance_data_view.sql

```diff
@@ -9,21 +9,29 @@ SELECT
     'fundName', fund.jsonb ->>'name',
     'fundDescription', fund.jsonb ->>'description',
     'fundStatus', fund.jsonb ->>'fundStatus',
-    'fundTags', fund.jsonb ->'tags' -> 'tagList',
+    'fundTags', jsonb_build_object('tagList', fund.jsonb -> 'tags' -> 'tagList'),
     'fundAcqUnitIds', fund.jsonb ->'acqUnitIds',
+    'ledgerId', ledger.id,
+    'ledgerCode', ledger.jsonb ->> 'code',
     'budgetId', budget.id,
     'budgetName', budget.jsonb ->>'name',
     'budgetStatus', budget.jsonb ->>'budgetStatus',
     'budgetInitialAllocation', budget.jsonb ->>'initialAllocation',
     'budgetCurrentAllocation', budget.jsonb ->>'allocated',
     'budgetAllowableExpenditure', budget.jsonb ->>'allowableExpenditure',
     'budgetAllowableEncumbrance', budget.jsonb ->>'allowableEncumbrance',
-    'budgetAcqUnitIds', budget.jsonb ->'acqUnitIds'
+    'budgetAcqUnitIds', budget.jsonb ->'acqUnitIds',
+    'groupId', groups.id,
+    'groupCode', groups.jsonb ->> 'code'
   ) as jsonb
 FROM ${myuniversity}_${mymodule}.fiscal_year
 LEFT OUTER JOIN ${myuniversity}_${mymodule}.ledger
-  ON ledger.fiscalyearoneid = fiscal_year.id
+    ON ledger.fiscalyearoneid = fiscal_year.id
 LEFT OUTER JOIN ${myuniversity}_${mymodule}.fund
-  ON fund.ledgerid = ledger.id
+    ON fund.ledgerid = ledger.id
 LEFT OUTER JOIN ${myuniversity}_${mymodule}.budget
-  ON fund.id = budget.fundid;
+    ON fund.id = budget.fundid
+LEFT OUTER JOIN ${myuniversity}_${mymodule}.group_fund_fiscal_year
+    ON fund.id = (group_fund_fiscal_year.jsonb ->> 'fundId')::uuid
```

Comment:
can be used just group_fund_fiscal_year.fundId, it is indexed field
![image](https://github.com/user-attachments/assets/6a040aa2-2f80-40ea-ae86-66cf6cf2d091)
The same for next join, group_fund_fiscal_year has groupId

Reply:
Thanks, updated both